usaspending_api/etl/tests/integration/test_load_to_delta.py

```diff
@@ -94,6 +327,18 @@ def _verify_delta_table_loaded(spark: SparkSession, delta_table_name: str, s3_bu
     assert equal_datasets(dummy_data, received_data, TABLE_SPEC[delta_table_name]["custom_schema"])
 
 
+def create_and_load_all_delta_tables(spark: SparkSession, s3_bucket: str):
+    non_aggregate_table_specs = {
+        key: val
+        for key, val in TABLE_SPEC.items()
+        if val.get("source_table") is not None and key == val.get("source_table")
+    }
+    for dest_table in non_aggregate_table_specs:
+        call_command("create_delta_table", f"--destination-table={dest_table}", f"--spark-s3-bucket={s3_bucket}")
+        call_command("load_table_to_delta", f"--destination-table={dest_table}")
+    create_ref_temp_views(spark)
```

Comment:
Might need to / want to see how we can incorporate `--alt-db` at least, and maybe also `--alt-table` in this. It could be difficult though. If this is just used by unit tests, may be unnecessary.

Reply:
Will do